# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp
# Commit: 754436d8b3a2
# Full Hash: 754436d8b3a2023b90eb247b3a70335e74f2fdf6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-09 09:14:23
# Regressor Bug: 1633628
# File Overlap Count: 3
# Description:
#   Bug 1633628 - Vender: Don't use ClearView if we previously used dual source blending on Intel gen6. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74137
# ==============================================================================

diff -r f8fd6f0a5102 -r 754436d8b3a2 gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp	Fri May 08 18:39:28 2020 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp	Fri May 08 19:28:53 2020 +0000
@@ -2435,7 +2435,9 @@
             // Haswell drivers occasionally corrupt (small?) (vertex?) texture data uploads for
             // 128bit formats.
             features->setDataFasterThanImageUploadOn128bitFormats.enabled = false;
-        }
+        } else if (IsSandyBridge(adapterDesc.DeviceId)) {
+	    features->emulateClearViewAfterDualSourceBlending.enabled = true;
+	}
     }
 
     if (IsAMD(adapterDesc.VendorId))